tests/scheduler/genetic/fixtures.py

```diff
@@ -48,15 +46,16 @@ def setup_toolbox(setup_default_schedules) -> tuple:
             resources_border[0, worker_index, work_index] = req.min_count
             resources_border[1, worker_index, work_index] = req.max_count
 
-    return (create_toolbox_and_mapping_objects(setup_wg,
-                                               setup_contractors,
-                                               setup_worker_pool,
-                                               size_of_population,
-                                               mutate_order,
-                                               mutate_resources,
-                                               setup_default_schedules,
-                                               rand,
-                                               work_estimator=work_estimator,
-                                               landscape=setup_landscape_many_holders,
-                                               verbose=False)[0], resources_border,
+    return (create_toolbox(setup_wg,
```

Comment:
it schould be create_toolbox_and_mapping_objects
or
from sampo.scheduler.genetic.schedule_builder import create_toolbox_and_mapping_objects as create_toolbox